package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@cloudogu/dogu-integration-test-library",
+  "version": "0.1.6-development",
```

Comment:
If I understand correctly, we will make a release afterwards, the "-development" should then disappear?